lib/tools.js

```diff
@@ -2006,18 +2006,16 @@ function formatAliasValue(sourceObj, targetObj, state, logger, logNamespace) {
         }
     }
 
-    // auto-scaling, only if val not null and unit for target or source is %
+    // auto-scaling, only if val not null and unit for target (x)or source is %
     if (((targetObj && targetObj.alias && !targetObj.alias.read) || (sourceObj && sourceObj.alias && !sourceObj.alias.write)) && state.val !== null) {
-        if (targetObj &&
-            targetObj.type === 'number' &&
-            targetObj.max !== undefined &&
-            targetObj.min !== undefined &&
-            sourceObj &&
-            sourceObj.max !== undefined &&
-            sourceObj.min !== undefined &&
-            (targetObj.unit === '%' || sourceObj.unit === '%')) {
-            const val = (state.val - sourceObj.min) / (sourceObj.max - sourceObj.min);
-            state.val = (targetObj.max - targetObj.min) * val + targetObj.min;
+        if (targetObj && targetObj.type === 'number' && targetObj.unit === '%' &&
+            sourceObj && sourceObj.min !== undefined && sourceObj.max !== undefined) {
```

Comment:
`&& sourceObj.type === 'number' && sourceObj.unit !== '%'` ?

Reply:
yes deleted it in between while rewriting ;-)